symLogging.py

```diff
@@ -1,29 +1,54 @@
 import sys
 import threading
 import time
+import logging
+import os.path
+from logging.handlers import RotatingFileHandler
 
 gEnableTracing = False
+logger = None
 
 # Whether to pause and launch debugger at CheckDebug breakpoints
 gDebug = False
 
-def SetTracingEnabled(isEnabled):
-  global gEnableTracing
+
+def SetTracingEnabled(isEnabled, logOptions):
+  global gEnableTracing, logger
   gEnableTracing = isEnabled
 
+  filepath = logOptions.get('logPath', '/var/log/snappy')
```

Comment:
this server will be run on *nix servers and developers' Windows machines as well, so hard-coding paths like this isn't ideal. Either check for OS type and use different paths for each OS, or use tempfile.gettempdir to log to a temporary directory if no logging dir is configured in the .ini